File: c/classical/AE00.c
/* http://www.spoj.com/problems/AE00/ */
#include<stdio.h>
#include<math.h>

int main() {
  int n, i, j, answer = 0;
  scanf("%d", &n);
  
  /* counting rectangles */
  for(i = 1; i <= sqrt(n); i++)
    for(j = i+1; i*j <= n; j++)
      answer++;
 
  /* adding possible amount of squares */
  answer += sqrt(n);

  printf("%d", answer);
  return 0;
}

File: cs/classical/AE00.cs
// http://www.spoj.com/problems/AE00/
using System;

namespace AE00 {
    class AE00 {
        static void Main(string[] args) {
            var n = Convert.ToInt32(Console.ReadLine());
            var answer = 0;
            for (var i = 1; i <= n; i++) {
                for (var j = i; j <= n; j++) {
                    if (i * j > n)
                        break;
                    answer++;
                }
            }
            Console.WriteLine(answer);
        }
    }
}